packages/itwin/tree-widget/src/tree-widget-react/components/trees/common/Utils.ts

```diff
@@ -24,8 +26,9 @@ export type FunctionProps<THook extends (props: any) => any> = Parameters<THook>
  * Disables display of all given models.
  * @public
  */
-export async function hideAllModels(models: string[], viewport: Viewport) {
-  viewport.changeModelDisplay(models, false);
+export async function hideAllModels(models: string[], viewport: Viewport | TreeWidgetViewport) {
+  const treeWidgetViewport = isTreeWidgetViewport(viewport) ? viewport : createTreeWidgetViewport(viewport);
```

Comment:
this gets repeated a lot, maybe `createTreeWidgetViewport` could take `Viewport | TreeWidgetViewport` and take care of this? then we could simply do this:
```
createTreeWidgetViewport(viewport).changeModelDisplay({ modelIds: models, display: false });
```